xivo_fetchfw/package.py

```diff
@@ -232,7 +228,7 @@ def _uninstall_pkg(self, installed_pkg, root_dir):
             for removed_path in remove_paths(installed_paths, root_dir):
                 removed_paths.append(removed_path)
         except Exception as e:
-            logger.error("Error while removing files of pkg %s: %s" % (pkg_id, e))
+            logger.error(f"Error while removing files of pkg {pkg_id}: {e}")
```

Comment:
I know that it was already used wrongly in the previous version, but a logger should be lazy for string interpolation. Usually we let the logger library do the job.
In this case the syntax to do so would be:
```python3
logger.error("Error while removing files of pkg %s: %s", pkg_id, e)
```

Reply:
Oh ok, any particular reason? They both only happen if they reach that point in the code. I've seen both everywhere. Just curious.